src/main/java/cat/udl/eps/softarch/academicrecruit/config/WebSecurityConfig.java

```diff
@@ -34,6 +34,10 @@ protected void configure(HttpSecurity http) throws Exception {
                 .antMatchers(HttpMethod.GET, "/jobApplications/*").authenticated()
                 .antMatchers(HttpMethod.POST, "/jobApplications/*").denyAll()
 
+                .antMatchers(HttpMethod.POST, "/applicants").authenticated()
```

Comment:
Now, anyone authenticated can create Applicants. In the future, when we can associate Applicants to JobApplications, we will need to add additional checks so it is not possible to add applicants to applications the user is not involved in. And maybe restrict this feature just to administrators or the secretary of the corresponding application...